WordPress/src/main/java/org/wordpress/android/ui/sitecreation/misc/SiteCreationTracker.kt

```diff
@@ -57,20 +56,6 @@ class SiteCreationTracker @Inject constructor(val tracker: AnalyticsTrackerWrapp
         )
     }
 
-    fun trackSegmentsViewed() {
```

Comment:
Thank you for removing this code @ovitrif 🙇

Reply:
Thank you @antonis 🙇 
There's even more code related to `segments` that could be removed in the future. For this PR limited my changes to events tracking, because the PR is about it 😄